trunk/src/app/srs_app_gb28181.cpp

```diff
@@ -22,6 +22,10 @@
 #include <srs_app_pithy_print.hpp>
 #include <srs_app_rtmp_conn.hpp>
 #include <srs_protocol_raw_avc.hpp>
+#include <srs_app_server.hpp>
+#include <srs_protocol_json.hpp>
+#include <srs_app_http_api.hpp>
+#include <srs_app_statistic.hpp>
```

Comment:
Is this include, `srs_app_statistice.hpp`, a useless one?

Reply:
It's not being used here at the moment, but it will be needed for adding statistical functionality later on. Let's keep it for now.